crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -115,8 +115,8 @@ def start_step(step_name)
           raise Crowbar::Error::StartStepExistenceError.new(step_name)
         end
         if running?
-          @logger.warn("Some step is already running.")
-          raise Crowbar::Error::StartStepRunningError.new
+          @logger.warn("Step #{current_step} is already running.")
+          raise Crowbar::Error::StartStepRunningError.new(step_name = current_step)
```

Comment:
@Itxaka are you sure this is correct behavior? `running?` without parameters checks *any* step. To get more detailed info you'd need to first call `running?(step_name)`, return with your message and later fallback to the previous code for any step.

PS. if the constructor has one argument, I agree with @jsuchome about simple arg passing.